optimum/exporters/onnx/model_patcher.py

```diff
@@ -1381,3 +1425,58 @@ def __init__(
             model_kwargs["dataset_index"] = torch.tensor(0, device=model.device)
 
         super().__init__(config, model, model_kwargs)
+
+
+def qwen3_moe_forward_patched(self, hidden_states: torch.Tensor) -> torch.Tensor:
```

Comment:
would you mind adding a link to the original code (using v4.53-release branch or similar) + highlight modifications